tensorflow_ranking/python/model.py

```diff
@@ -396,6 +402,94 @@ def _compute_logits_impl(self, context_features, example_features, labels,
     return logits
 
 
+class _MultiTaskGroupwiseRankingModel(_GroupwiseRankingModel):
```

Comment:
This is a very nice proposal. Indeed, we had a proposal internally and have been working on some implementation. Your PR is a very good reference for us to build a polished version.

Since this touches the core code, we would like this to be as concise as possible. We will also have multiple small changes to achieve the goal of multi-task learning.

Stayed tuned.